src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/Scripts/AzureDiskEncryptionPreRequisiteSetup.ps1

```diff
@@ -1,4 +1,6 @@
-﻿
+﻿#Requires -Module AzureRM.Resources
```

Comment:
@SudhakaraReddyEvuri 😀

Reply:
@cormacpayne  thanks for your suggestion  :)